File: padroesComportamentais/padrao-de-projeto-iterator/java/src/main/java/Produto.java
public class Produto {

    private String nome;
    private Float valor;

    public String getNome() {
        return nome;
    }

    public Produto setNome(String nome) {
        this.nome = nome;
        return this;
    }

    public Float getValor() {
        return valor;
    }

    public Produto setValor(Float valor) {
        this.valor = valor;
        return this;
    }
}

File: padroesComportamentais/padrao-de-projeto-iterator/php/src/Produto.php
<?php

namespace Iterator\Php;

class Produto
{
    private string $nome;
    private float $valor;

    public function getNome(): string
    {
        return $this->nome;
    }

    public function setNome(string $nome): Produto
    {
        $this->nome = $nome;
        return $this;
    }

    public function getValor(): float
    {
        return $this->valor;
    }

    public function setValor(float $valor): Produto
    {
        $this->valor = $valor;
        return $this;
    }

}